widgets/tabs/tpl/default.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @var array $args
+ * @var array $instance
+ * @var array $tabs
+ * @var array $initial_tab_index
+ */
+
+$last_index = count( $tabs ) - 1;
+
+echo $args['before_title'] . $instance['title'] . $args['after_title'];
+?>
+<div class="sow-tabs">
+	<div class="sow-tabs-tab-container">
+	<?php foreach ( $tabs as $i => $tab ) : ?>
+		<?php
```

Comment:
Similar to my feedback for the accordion, I'd really like to see this in `get_template_vars` and possibly with a filter.